README.md

```diff
@@ -4,10 +4,11 @@
 
 Create certificates for HTTPS
 ```bash
-chmod u+x ./scripts/generate-certificates.sh
-./scripts/generate-certificates.sh
+./scripts/pontsun generate-cert
+```
+You need to add the generated certificate `etc/certificates/docker.rootCA.crt` to your browser authorities and trust related websites.
```

Comment:
This should probably be replaced by `etc/certificates/pontsun.rootCA.crt` for consistency.

Reply:
fixed that. you're right of course